backends-clickhouse/src/main/scala/io/glutenproject/execution/CHHashAggregateExecTransformer.scala

```diff
@@ -31,7 +31,7 @@ import org.apache.spark.sql.execution.adaptive.QueryStageExec
 import org.apache.spark.sql.execution.aggregate.BaseAggregateExec
 import org.apache.spark.sql.execution.exchange.Exchange
 
-import com.google.protobuf.Any
+import shaded.com.google.protobuf.Any
```

Comment:
It looks several other source code files also need such change. Right? Please search "import com.google.protobuf.Any" in the project.

Reply:
I think we don't need to change in other places (all in gluten-core), based on some experiments, the shade plugin seems only relocate classes when the package is packed as a jar....so in this case

(1) we don't need to use shaded.* in gluten-core, since when the tests run, classes haven't been packed
(2) we need to relocate in gluten-velox and gluten-ch , since they refer to gluten-core as a dependency which has been packed as a jar, so they have to use shaded to explicitly refer to the newer protobuf brought by gluten-core instead of Spark

Reply:
Can we use io.glutenproject.shaded as prefix?